src/Repository/EventRegistrationRepository.php

```diff
@@ -188,11 +188,44 @@ public function countEventParticipantsInReferentManagedArea(Adherent $referent,
             ->groupBy('yearmonth')
         ;
 
-        $query = RepositoryUtils::addStatstFilter($filter, $query);
+        if ($filter) {
+            $query = RepositoryUtils::addStatstFilter($filter, $query);
+        }
 
         return RepositoryUtils::aggregateCountByMonth($query->getQuery()->getArrayResult(), 'event_participants');
     }
 
+    public function countEventParticipantsInReferentManagedAreaInAtLeastOneCommittee(Adherent $referent, StatisticsParametersFilter $filter = null, int $months = 5): array
+    {
+        $this->checkReferent($referent);
+
+        $query = $this->createQueryBuilder('event_registrations')
+            ->select('DISTINCT event_registrations.emailAddress, COUNT(DISTINCT event_registrations) AS count, YEAR_MONTH(event.beginAt) as yearmonth')
```

Comment:
why do you need the email address? It's useless with `->groupBy('yearmonth')` no?

Reply:
I do agree but perhaps we are missing something. @illusionOfParadise What do you think ?

Reply:
One person with the same email adresse can be registered to several events in one month and we need to take it into account only once.

Reply:
In fact no, we talk in PM with @illusionOfParadise and we figured out that one person should be taken into account more than once.

Reply:
So this distinct is removed.